app/services/finance/stripe_charge_service.rb

```diff
@@ -75,4 +80,10 @@ def extract_payment_intent_data_from(response)
     payment_intent_data = (response.success? ? response_params : response_params.dig('error', Stripe::PaymentIntent::OBJECT_NAME)) || {}
     payment_intent_data if payment_intent_data['object'] == Stripe::PaymentIntent::OBJECT_NAME
   end
+
+  def charge_description
+    return PAYMENT_DESCRIPTION if invoice.blank?
+
+    "#{invoice&.from&.name} #{PAYMENT_DESCRIPTION} #{invoice&.friendly_id}".strip
```

Comment:
The guard clause ensures that, by the point we get to this line, an invoice exists. Because of that, we can remove the safe navigation operator 🎉 

```suggestion
    "#{invoice.from&.name} #{PAYMENT_DESCRIPTION} #{invoice.friendly_id}".strip
```

**MAYBE** from is always present due to a validation or something else? If that's the case, we may remove the last safe navigator as well.